bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -139,14 +139,14 @@ spec:
                         - linux
               containers:
               - args:
-                - --secure-listen-address=0.0.0.0:8443
+                - --secure-listen-address=0.0.0.0:9443
                 - --upstream=http://127.0.0.1:8080/
                 - --logtostderr=true
                 - --v=0
                 image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
                 name: kube-rbac-proxy
```

Comment:
What is the intent to have kube-rbac-proxy?

Reply:
This will come into picture when we want to expose operator metrics to console. As for the port change, the issue is that there is some conflict with 8443 with the operator also listening on the same port by default (https://github.com/openshift/library-go/blob/8ce03fe3cf2269c39e9fbd04473053a0925d0b1f/pkg/controller/controllercmd/cmd.go#L62). We can get around to fixing this post this OLM PR.